src/scheduler/entrypoint.sh

```
@@ -51,6 +51,32 @@ DATABASE_URI=${DATABASE_URI:-sqlite:////var/lib/bunkerweb/db.sqlite3}
+else
+	log "ENTRYPOINT" "⚠️" "Failed to parse DATABASE_URI (error), using defaults"
+	type="(unknown)"
+	database="(none)"
+	host="(local)"
+	user="(none)"
+	ssl_enabled="no"
+	pem_path="(none)"
+fi
+db_ssl_enabled="$ssl_enabled"
+db_pem_path="$pem_path"
+db_port="$port"
```

Shellcheck (0.11.0)

[warning] port is referenced but not assigned.

(SC2154)